src/gamma.jl

```diff
@@ -20,9 +17,13 @@ function _gamma(x::Float64)
         w = muladd(w, evalpoly(w, s), 1.0)
         # avoid overflow
         v = x ^ muladd(0.5, x, -0.25)
-        y = v * (v / exp(x))
+        res = SQ2PI(T) * v * (v / exp(x)) * w
 
-        return SQ2PI(T) * y * w
+        if _x < 0
+            return π / (res * s)
```

Comment:
This seems to be the issue. We are dividing by `s` which is the polynomial coefficients.

Reply:
Also, the `s` looks to be type unstable considering it is defined as a Float64 in the top half.

Reply:
ah. Variable names strike again.